.gitlab-ci.yml

```diff
@@ -81,14 +80,13 @@ build:raspios:
 
 build:acceptance-testing:
   tags:
-    - hetzner-amd-beefy
+    - mender-qa-worker-generic-light
```

Comment:
I would keep the private runner for every other job but `build:prepare:raspios` and then also create a task to find a different method to do this. We don't really need to mount it, we can use debugfs or other tools to write to/from the image.

Reply:
I reverted also the other jobs because of docker hub rate limit issues (again), but it seems a typo in the config.toml:
```
environments = ["DOCKER_AUTH_CONFIG=
# That should be
environment = ["DOCKER_AUTH_CONFIG=
```
I fixed that, so let's revert just the `build:prepare:raspios` and keep the other jobs with the private runner.